src/main/commands/commandManager.ts

```diff
@@ -37,6 +39,11 @@ export class CommandManager implements ExtensionComponent {
         // Local state
         this.registerCommand(context, 'jfrog.issues.file.open', file => ScanUtils.openFile(file));
         this.registerCommand(context, 'jfrog.issues.file.open.location', (file, fileRegion) => ScanUtils.openFile(file, fileRegion));
+        this.registerCommand(context, 'jfrog.issues.select.node', item => this._treesManager.selectItemOnIssuesTree(item));
+        this.registerCommand(context, 'jfrog.issues.file.open.applicable', async (file, fileRegion, details) => {
+            await ScanUtils.openFile(file, fileRegion);
```

Comment:
Let's not wait here.

Reply:
if we remove waiting here the file and the webpage will open on the same ViewColumn.
the open file command opens on ViewColumn.active and the webpage will open on ViewColumn.Beside
if we don't wait it will open the webpage on a new column and it will become the active that the file will open on